contributing/documentation/standards.rst

```diff
@@ -160,6 +172,7 @@ English Language Standards
 .. _`the Sphinx documentation`: http://sphinx-doc.org/rest.html#source-code
 .. _`Twig Coding Standards`: http://twig.sensiolabs.org/doc/coding_standards.html
 .. _`American English`: http://en.wikipedia.org/wiki/American_English
+.. _`American English Oxford Dictionary`: http://www.oxforddictionaries.com/
```

Comment:
The problem is that I can't find a direct link to the American English dictionary. The given URL is correct (at least for me) but you have to do several things to look for a word:

![dictionary](https://cloud.githubusercontent.com/assets/73419/6542514/d51f1870-c4fa-11e4-9ac3-d8fd92314049.gif)